➕  aggiunta capitolo conferenze (#266)
Il-Libro-Open-Source/book: docs/it/formazione/conferenze.md

Reviewer: Direi top, aggiungerei solo alle conferenze anche queste

- Open Source Day (https://osday.dev)
- Come To Code (https://www.cometocode.it/)
- Coderful (https://www.coderful.io/)
- WeAreDevelopers World Congress (https://www.wearedevelopers.com/world-congress)
- SoCraTes (https://www.socrates-conference.it/)
- Working Software (https://www.agilemovement.it/workingsoftware/)

Si potrebbe inoltre citare https://eventitech.it/ che fa un buon lavoro di collettore di tutte le tech conf.

---
layout: default
title: Le conferenze: il cuore pulsante dell'open source
parent: Formazione
nav_order: 4
---

<!-- prettier-ignore-start -->
# Le conferenze: il cuore pulsante dell'open source
{: .no_toc }

- TOC
{:toc}

<!-- prettier-ignore-end -->

_Aggiornato al 2025-06-18_

Nel mondo open source, le conferenze non sono solo occasioni per presentare codice o mostrare progetti: sono il luogo dove le idee prendono forma, le comunità si rafforzano, e le collaborazioni nascono o si consolidano.

Sono momenti di incontro reale in un universo spesso distribuito e virtuale.

Le conferenze sono importanti per:

- **Comunità**: favoriscono la creazione e il rafforzamento delle relazioni personali, elemento chiave per progetti collaborativi.

- **Formazione continua**: offrono workshop, talk tecnici e aggiornamenti su tecnologie emergenti.

- **Inclusività e accessibilità**: molte conferenze open source sono gratuite o a basso costo, con opzioni ibride (in presenza ).

- **Spirito di condivisione**: si promuovono valori come trasparenza, collaborazione e meritocrazia.

Di seguito si è voluto mettere un elenco delle conferenze italiane e internazionali.

> Non è certo un elenco completo, ma questo capito fa parte de "Il libro Opensource" perciò siamo aperti ad aggiunte! ^\_^

Reviewer 2: ```suggestion
Non è certo un elenco completo, ma questo capitolo fa parte de "Il Libro Open Source" perciò siamo aperti ad aggiunte! ^\_^
```

| Product Management Day     | Bologna       | Conferenza dedicata al Product Management, con focus su sviluppo integrato e crescita orientata al prodotto.  | productmanagementday.com                | GrUSP                |
| devsecopsDay               | Bologna       | Evento italiano dedicato alle pratiche di sicurezza in ambito DevOps.                                         | devsecopsday.it                         | GrUSP                |
| ContainerDay               | Bologna       | Conferenza su tecnologie container, Docker, Kubernetes e microservizi.                                        | containerday.it                         | GrUSP                |
| LaravelDay                 | Verona        | Evento dedicato al framework Laravel e al suo ecosistema.                                                     | laravelday.it                           | GrUSP                |
| AngularDay                 | Verona        | Conferenza su Angular e tecnologie correlate, con sessioni in lingua inglese.                                 | angularday.it                           | GrUSP                |

> | AWS Summit                 | Milano        | Conferenza di AWS sulle notiva del RE-invent                                                                  | aws.amazon.com/it/events/summits/milano | Amazon Web Services  |

Reviewer 2: ```suggestion
| AWS Summit                 | Milano        | Conferenza di AWS sulle novità del RE-invent                                                                  | aws.amazon.com/it/events/summits/milano | Amazon Web Services  |
```

| Codemotion Tech Conference | Roma / Milano | Una delle principali conferenze tech in Europa su sviluppo software, AI, cloud, DevOps, frontend e community. | codemotion.com                          | Codemotion           |
| Digital Design Days | Milano / Palermo | evento internazionale su innovazione, design digitale, creatività, tecnologia e ispirazione. | ddd.live                         | DDD srl           |
| Open Source Day                | Roma / online     | Evento sull’open source con talk tecnici e casi d’uso per promuovere collaborazione e innovazione. | osday.dev                        | DevRomagna + GrUSP       |
| Come To Code                   | Bologna           | Conferenza per sviluppatori con talk e workshop su coding, tecnologie e buone pratiche.            | cometocode.it                    | LinkMe + ideato          |
| Coderful                       | Online / Italia   | Percorso per sviluppatori con eventi, sfide tecniche e occasioni di crescita professionale.        | coderful.io                      | Coderful Srl             |

> | SoCraTes                       | Rimini            | Unconferenza su qualità del software, sviluppo consapevole e scambio peer-to-peer.                 | socrates-conference.it           | SoCraTes Italy Community |

Reviewer 2: ```suggestion
| SoCraTes                       | Rimini            | Una conferenza su qualità del software, sviluppo consapevole e scambio peer-to-peer.                 | socrates-conference.it           | SoCraTes Italy Community |
```
